Packages/com.unity.multiplayer.samples.coop/Utilities/Testing/TestUtilities.cs

```diff
@@ -0,0 +1,163 @@
+using System.Collections;
+using UnityEngine;
+using NUnit.Framework;
+using Unity.Netcode;
+using UnityEngine.EventSystems;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+namespace Unity.Multiplayer.Samples.Utilities
+{
+    public abstract class TestUtilities
+    {
+        const float k_MaxSceneLoadDuration = 10f;
+
+        /// <summary>
+        /// Finds an active Button GameObject by name. If Button component is present on GameObject, it will be clicked.
+        /// </summary>
+        /// <param name="name"> Name of Button GameObject </param>
+        public static void ClickButtonByName(string name)
```

Comment:
after our discussions, is this still needed?

Reply:
Correct. This has been removed.